apps/nrm/behaviours/next-steps/page-custom-next-steps.js

```diff
@@ -227,6 +227,22 @@ module.exports = {
       nextStep = '/nrm/confirm';
     }
 
+    return nextStep;
+  },
+  checkYourAnswersSoFar: (req) => {
+    let nextStep = '/nrm/start';
+
+    const savedData = req.sessionModel.get('report-read-success');
+
+    if (savedData) {
+      // get the last saved page - only caveat is that the user would have already answered this question
+      // ideally we would like to send them to the subsequent page
+      const visitedPages = savedData.visited_pages.split(',');
+      const getLastSavedPage = visitedPages[visitedPages.length - 1].trim();
```

Comment:
when they click save, it should really save the form data but we can do that later